glossary.yml

```
@@ -4575,6 +4575,15 @@
       A variable whose value is automatically updated when some other value or values
       change. Reactive variables are used extensively in [Shiny](#shiny).
 
+- slug: readme
+  ref:
+    - markdown
+    - vignette
+  en:
+    term: "README"
+    def: >
+      A file which contains the important information about a project or software package.
```

This matches the style of other definitions (e.g. Makefile) better.

```suggestion
      A plain text file containing important information about a project or software package.
```